tests/backend/access.c

```diff
@@ -4,7 +4,7 @@
 
 #include <gio/gio.h>
 
-#include "src/xdp-impl-dbus.h"
+#include "xdp-impl-dbus.h"
```

Comment:
This would be unnecessary with `include_directories : common_include_directories`.

Reply:
true, but otoh it seems like a more correct approach to have the include directory-less and adjust the include path as necessary? Even for the autotools build.

Reply:
I don't have strong opinions on this, and don't see any problem in the approach taken here